install/kubernetes/cilium/charts/hubble/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: hubble
+  namespace: {{ .Release.Namespace }}
+{{- if .Values.ui.enabled }}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: {{ .Release.Namespace }}
+  name: hubble-ui 
+{{- end }}
```

Comment:
Some of the files are missing a new line at the end of the file.

Reply:
Done